file_store/src/file_store.rs

```diff
@@ -198,6 +204,25 @@ impl FileStore {
             })
             .boxed()
     }
+
+    pub fn stream_files<A, B, F>(&self, file_type: F, after: A, before: B) -> Stream<FileData>
+    where
+        F: Into<FileType> + Copy,
+        A: Into<Option<DateTime<Utc>>> + Copy,
+        B: Into<Option<DateTime<Utc>>> + Copy,
+    {
+        let client = self.client.clone();
+        let bucket = self.bucket.clone();
+
+        self.list(file_type, after, before)
```

Comment:
is this thing going to return an error and the file on which the error occurred ?